package.json

```diff
@@ -24,6 +24,7 @@
     "test:try-one": "yarn workspace ember-data test:try-one",
     "test:docs": "yarn workspace ember-data test:docs",
     "test:encapsulation": "lerna run --scope='*-encapsulation-test-app' test",
+    "test:encapsulation:serializer": "yarn workspace serializer-encapsulation-test-app test",
```

Comment:
for local dev you can run `ember test` in the directory directly. For CI the `test:encapsulation` command already runs this. Let's remove this.

Reply:
done.